### DISCUSSION TITLE
How to "debug" email sending while using Weblate with Docker

### DISCUSSION THREAD

**Discussion by sowinski** [WeblateOrg/weblate#9221] [Q&A]
Hi,

I installed Weblate 4.17 with the docker image. 
Unfortunately I can not send E-Mails but I don't know why.

I set the ENV correctly. In the docker log I can not see any handy information.

1.) Can I check in the Django/Weblate Admin if my settings from my ENV is adopted correctly?
2.) Is there a log file with more insides?

Thank you :)

**Comment by nijel** [comment]
See https://docs.weblate.org/en/latest/contributing/debugging.html#debug-mails and https://docs.weblate.org/en/latest/contributing/debugging.html#weblate-logs
